### DISCUSSION TITLE
How to install APK?

### DISCUSSION THREAD

**Discussion by michaelbelcik** [atsushieno/android-audio-plugin-framework#203] [General]
Tried installing the APK from latest version

I read the README
[We have some hacky "AAP APK Installer"]
I navigated to this page and installed the most recent APK, but it says unable to install. Not very intelligent in this, doing something wrong? I can't wait to try a vst even with gui not fully supported

DO I need other files along with the vst to run it?

**Comment by atsushieno** [comment]
Whoa, I'm sorry, didn't notice this discussion post until now.
I'm sure you hit various AAP APK Installer issues (like, download size limitation due to GitHub API for Java, APK metadata mismatch in its app catalog, etc.). Your experience would not be totally due to your problem.

Last month I made a lot of changes and today made a new release which vastly improves the installer experience and fixed a lot of issues so that I could installed them on my device. It's still hacky, but way more functional than earlier. If you are still interested, please give it a try [via DeployGate](https://deploygate.com/organizations/atsushieno/platforms/android/apps/org.androidaudioplugin.aapapkinstaller) or [from source](https://github.com/atsushieno/android-ci-package-installer).
